packages/harmony/src/components/text-link/TextLink.tsx

```diff
@@ -47,6 +56,7 @@ export const TextLink = (props: TextLinkProps) => {
       css={{
         color: variantColors[variant],
         textDecoration: 'none',
+        transition: 'none',
```

Comment:
Be declarative about this because currently web client passes a global 0.3s ease to all <a>